### PR TITLE
und-1088

### PR DESCRIPTION
**Pull request by Ashish-srij**
Header issues fixed

### CODE REVIEW

File: stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/mega-menu.scss [hunk 1]
```diff
@@ -189,12 +146,65 @@
...
+      @extend %mega-animation;
+      -webkit-animation-fill-mode: forwards;
+      animation-fill-mode: forwards;
+  
+      > div{
+        -webkit-animation-delay: .3s;
+        animation-delay: .3s;
+        -webkit-animation-duration: .3s;
+        animation-duration: .3s;
+        -webkit-animation-fill-mode: forwards;
```
Comment (shishirphadke) [inline stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/mega-menu.scss:162]: @Ashish-srij We have created a mixin for the animation. Why are we not using that? Just use it simply in the following way:
Eg.
```
@include animation(transition-opacity forwards 0.3s 0.3s);
```
image

File: stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/mega-menu.scss [hunk 2]
```diff
@@ -189,12 +146,65 @@
...
+      animation-delay: 0s;
+      -webkit-animation-duration: 0s;
+      animation-duration: 0s;
+  
+      > div{
+        opacity: 1;
+        -webkit-animation-delay: 0s;
+        animation-delay: 0s;
+        -webkit-animation-duration: 0s;
+        animation-duration: 0s;
```
Comment (shishirphadke) [inline stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/mega-menu.scss:188]: Similarly here and above.

Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/mega-menu.scss]: @shishirphadke Done.

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/global-header.scss [hunk 3]
```diff
@@ -3,167 +3,153 @@
...
+          height: 4.6875rem;
+        }
+      }
+      ul {
+        position: absolute;
         left: 50%;
-        position: fixed;
+        width: 50%;    
+        top: 50%;
+        transform: translateY(-50%);
```
Comment (shishirphadke) [inline stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/global-header.scss:50]: @Ashish-srij Change this to `@include transform(translateY(-50%));`

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/global-header.scss [hunk 4]
```diff
@@ -3,167 +3,153 @@
...
-      padding: 0;
+      .icon-globe {
+        background: none;
+        border: 0;
+        display: inline-flex;
+        padding: 0;
+        @include devicebreak(mediumonlytab) {
+          display: none;
+        }
+      }
```
Comment (shishirphadke) [inline stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/global-header.scss:120]: @Ashish-srij These are interactive buttons and need to have a cursor pointer. Right now the cursor is default. Please change.

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/global-header.scss [hunk 5]
```diff
@@ -3,167 +3,153 @@
...
+
+      .icon-search {
+        background: none;
+        border: 0;
+        display: inline-flex;
+        margin-left: $spacing-16;
+        margin-right: $spacing-12;
+        padding: 0;
+      }
```
Comment (shishirphadke) [inline stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/global-header.scss:129]: Similarly here. Please add cursor pointer.

**Review by shishirphadke — CHANGES_REQUESTED** [review]

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/global-header.scss (hunk 5 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/global-header.scss:131]: @shishirphadke Done.

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/global-header.scss (hunk 4 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/global-header.scss:121]: @shishirphadke Done.

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/global-header.scss (hunk 3 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/global-header.scss]: @shishirphadke Done.

**Review by shishirphadke — APPROVED** [review]

**Pull request merged**